tests/test_wmb_download.py

```diff
@@ -1,7 +1,7 @@
-from crmprtd.wmb.download import logging_setup
+from crmprtd.wmb import setup_logging
```

Comment:
Looks like this PR drops the test coverage from 62% down to 55% so presumably, there's a lot of untested code in here. Can you start putting together a set of tests for your align phase? That *may* help you to decompose it a bit more to think of it in testable units.